extensions/amp-user-notification/0.1/amp-user-notification.js

```diff
@@ -308,14 +333,17 @@ export class AmpUserNotification extends AMP.BaseElement {
   /**
    * Hides the current user notification and invokes the `dialogResolve_`
    * method. Removes the `.amp-active` class from the element.
+   *
+   * @param {boolean=} opt_forceNoPersist If true, dismissal won't be persisted
+   * regardless of 'data-persist-dismissal''s value
    */
-  dismiss() {
+  dismiss(opt_forceNoPersist = false) {
```

Comment:
Danger! This requires auditing every call to `dismiss`, since there may have been a values passed into it through callbacks (the `then`s are already doing it in this PR).